src/Error.cc

```diff
@@ -0,0 +1,73 @@
+/**
+ * \file   Error.cc
+ * \author Lars Fröhlich, Jannik Wöhnert
```

Comment:
Nah. ;)

Reply:
I believe we should drop all those `\author` lines anyhow; I hate them.

I also hate the copyright-year line too btw, but to a lesser extend. 

Can we not say something like `\author The project authors`or something? :grimacing:

Reply:
I honestly have no clue. I think these author lists have some legal meaning to point out which individuals worked on the code at which point in time (to figure out if their copyright is still valid at some point in the future). So in theory this should probably look like this:
```
\copyright
1999 D. Duck
2000-2012, 2015 M. Mouse
2020 Tick, Trick, Track
```
But... :shrug:

Reply:
... in any case, I did not write a line of this. :-D